lua/obsidian/picker_utils.lua

```diff
@@ -16,12 +16,25 @@ M.telescope_mappings = function(map, client, initial_query)
       require("telescope.actions").close(prompt_bufnr)
       client:command("ObsidianNew", { args = query })
     end,
+
+    obsidian_insert_link = function(prompt_bufnr)
+      local selected_path = require("telescope.actions.state").get_selected_entry().path
+      local vault_relative_path = client:vault_relative_path(selected_path)
+      require("telescope.actions").close(prompt_bufnr)
+      vim.api.nvim_put({ "[](" .. vault_relative_path .. ")" }, "", false, true)
```

Comment:
We should allow users to customize the form of these links. We already have some relevant configuration options in the `completion` settings, such as `completion.use_path_only`, `completion.prepend_note_path`, and `completion.prepend_note_id` (see https://github.com/epwalsh/obsidian.nvim/blob/7d6574dcb8c3c7051f263824f1fdfdb1798c8de2/lua/cmp_obsidian.lua#L58-L80 for how these are used). We may need another option (eg. `completion.preferred_link_style = "markdown" | "wiki"`) in this case to switch between markdown-style vs wiki-style links.